app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -342,23 +342,30 @@ def build_custom_toolbar_class(model, record, toolbar_result)
       toolbar.button_group(button_group[:name], button_group[:items])
     end
 
+    custom_button_add_related_buttons(model, record, toolbar)
+    toolbar
+  end
+
+  def custom_button_add_related_buttons(model, record, toolbar)
     # For Service, we include buttons for ServiceTemplate.
     # These buttons are added as a single group with multiple buttons
     if record.present?
```

Comment:
@lgalis do we need this check, i see record being passed in from the caller

Reply:
yes, record may not always be present